oauth2client/django_orm.py

```diff
@@ -46,12 +46,16 @@ def to_python(self, value):
             return None
         if isinstance(value, oauth2client.client.Credentials):
             return value
-        return pickle.loads(base64.b64decode(value))
+        return pickle.loads(base64.b64decode(smart_bytes(value)))
 
-    def get_db_prep_value(self, value, connection, prepared=False):
```

Comment:
@szopu I'm not a Django user, can you explain why you got rid of ` get_db_prep_value` and swapped out for `get_prep_value` and `value_to_string`?